p2p/simulations/http.go

```diff
@@ -629,6 +631,8 @@ func (s *Server) GetNode(w http.ResponseWriter, req *http.Request) {
 // DeleteNode deletes a node from the network
 func (s *Server) DeleteNode(w http.ResponseWriter, req *http.Request) {
 	node := req.Context().Value("node").(*Node)
+	// Node must be stopped before it can be deleted
+	s.network.Stop(node.ID())
```

Comment:
if node.Up() {
		s.network.Stop(node.ID())
                 log.info ....
	}

Reply:
I saw that the node has already been checked to see whether it has started in the `s.network.Stop` function, so from my perspective, this addition check seems redundant